kernel/packages/shared/profiles/sagas.ts

```diff
@@ -150,6 +161,7 @@ export function* initialLoad() {
       const catalog = collections
         .reduce((flatten, collection) => flatten.concat(collection.wearables), [] as Wearable[])
         .map(overrideBaseUrl)
+        .map(overrideSwankyRarity)
```

Comment:
Is this because of the wearable-api or stored profiles?

Reply:
This data comes from the wearable-api. The idea is to avoid issues in this transition.

Reply:
Nice! I've implemented the change, but not deployed (will do on Tuesday) in case you want to add an issue to remove this override later.

Reply:
Good point, will do that then 😬